native/src/main/java/io/ballerina/stdlib/oracledb/parameterprocessor/OracleDBStatementParameterProcessor.java

```diff
@@ -68,19 +67,13 @@ protected void setCustomSqlTypedParam(Connection connection, PreparedStatement p
             case Constants.Types.CustomTypes.VARRAY:
                 setVarray(connection, preparedStatement, index, value);
                 break;
-            case Constants.Types.CustomTypes.NESTED_TABLE:
```

Comment:
Same

Reply:
issue was created to track this removal https://github.com/ballerina-platform/ballerina-standard-library/issues/1665